src/hooks/transactions/useTransactionType.ts

```diff
@@ -69,12 +70,15 @@ export default function useTransactionType({ input, type, weightInput }: Props):
             break;
           case TransactionTypes.TRANSFER:
             if (receiverAddress) {
-              call = (await targetApi.tx.balances.transfer(receiverAddress, input)).toU8a();
+              const planck = 10 ** targetApi.registry.chainDecimals[0];
+              const [resInp] = evalUnits((parseFloat(input) * planck).toString());
+              const transferInput = resInp?.toString() || '0';
+              call = (await targetApi.tx.balances.transfer(receiverAddress, transferInput)).toU8a();
```

Comment:
Why do we calculate it here again? There should be only one place where this is performed.